src/main/webapp/assets/modules/salary-report/sr-similar-job.js

```diff
@@ -5,24 +5,19 @@ techlooper.directive("srSimilarJob", function (jsonValue, connectionFactory, $ti
     templateUrl: "modules/salary-report/sr-similar-job.tem.html",
     link: function (scope, element, attr, ctrl) {
       var timeToSends = $.extend(true, [], jsonValue.timeToSends);
-
-      scope.$watch("salaryReview", function() {
-        if (scope.state.editableSalaryReview) {
-          scope.jobAlert = $.extend(true, {}, scope.salaryReview);
-          scope.jobAlert.frequency = timeToSends[0].id;
-          delete scope.jobAlert.salaryReport;
-          delete scope.jobAlert.topPaidJobs;
-        }
-      });
-
-
+      var lastHideButton = undefined;
```

Comment:
It's not necessary to initialize 'lastHideButton' to 'undefined'.<br>'lastHideButton' is defined but never used.